libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -578,6 +578,11 @@ bool AP_Arming::rc_arm_checks(AP_Arming::Method method)
         check_failed(ARMING_CHECK_PARAMETERS, true, "Duplicate Aux Switch Options");
         check_passed = false;
     }
+    uint8_t flight_mode_channel;
+    if (rc().check_flight_mode_channel_conflict(flight_mode_channel)) {
```

Comment:
```suggestion
    if (rc().flight_mode_channel_conflicts_with_rc_option()) {
```

And just use `flight_mode_channel_number()` here.

Reply:
the function is private.
https://github.com/ArduPilot/ardupilot/blob/master/libraries/RC_Channel/RC_Channel.h#L546
Can I change it to public?

Reply:
Yes, that's all good.  We're hoping to remove the `FLTMODE_CH` and `MODE_CH` parameters from the vehicles in 4.2 - but a method to retrieve that channel number will still be good.

Reply:
I fixed it